mkosi/__init__.py

```diff
@@ -3135,6 +3135,25 @@ def run_preset_all(state: MkosiState) -> None:
         run(["systemctl", "--root", state.root, "preset-all"])
 
 
+def run_selinux_relabel(state: MkosiState) -> None:
+    selinux = state.root / "etc/selinux/config"
+    if not selinux.exists():
+        return
+
+    policy = run(["sh", "-c", f". {selinux} && echo $SELINUXTYPE"], text=True, stdout=subprocess.PIPE).stdout.strip()
```

Comment:
Wouldn't that be a good use of `shell=True`?

Reply:
Maybe but I wouldn't sweat it with just one usecase